projects/plugins/boost/app/modules/critical-css/DisplayCriticalCSS.php

```diff
@@ -0,0 +1,156 @@
+<?php
+/**
+ * Critical CSS frontend display handler.
+ *
+ * @package automattic/jetpack-boost
+ */
+
+namespace Automattic\Jetpack_Boost\Modules\Critical_CSS;
+
+/**
+ * Class DisplayCriticalCSS.
```

Comment:
Is this comment just here to silence a lint rule? It doesn't add anything.

Reply:
This shouldn't be there at all :)